go.mod

```diff
@@ -3,78 +3,108 @@ module github.com/ignite/cli
 go 1.19
 
 require (
-	cosmossdk.io/math v1.0.0-beta.4
+	cosmossdk.io/api v0.3.1 // indirect
```

Comment:
We should group indirect dependencies in a separate `require` group from the direct ones